wca-integration/src/main/java/eu/itesla_project/wca/WCAUtils.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.wca;
+
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.Properties;
+
+import eu.itesla_project.iidm.datasource.DataSource;
+import eu.itesla_project.iidm.datasource.GzFileDataSource;
+import eu.itesla_project.iidm.export.Exporters;
+import eu.itesla_project.iidm.network.Network;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+public final class WCAUtils {
+
+    private WCAUtils() {
+    }
```

Comment:
Would you like to have a new interface, with just this static method? But what would be the purpose of the interface (besides having the static method)? Who would/should implement it?
Or would you like to have this static method in an existing interface, WCA?